# Certain tests commented out for stability reasons:
# listview: Fails when building inside sandbox
# mvattrline: Fails on some testers boxes. Documented here: https://github.com/tstack/lnav/issues/911
# logfile: Flaky. Fails some times and not others. i.e. Race condition
# tui: Can hang the build
--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -355,14 +355,14 @@
 	test_cli.sh \
 	test_cmds.sh \
 	test_config.sh \
-	test_listview.sh \
+	#test_listview.sh \
 	test_meta.sh \
-	test_mvwattrline.sh \
+	#test_mvwattrline.sh \
 	test_grep_proc.sh \
 	test_grep_proc2 \
 	test_json_format.sh \
 	test_log_accel \
-	test_logfile.sh \
+	#test_logfile.sh \
 	test_reltime \
 	test_remote.sh \
 	test_scripts.sh \
@@ -376,7 +376,7 @@
 	test_sql_str_func.sh \
 	test_sql_time_func.sh \
 	test_sql_xml_func.sh \
-	test_tui.sh \
+	#test_tui.sh \
 	test_data_parser.sh \
 	test_pretty_print.sh \
     test_view_colors.sh \
